mithril-test-lab/mithril-devnet/devnet-mkfiles.sh

```diff
@@ -6,6 +6,7 @@ set -e
 if [[ -n $DEBUG ]]; then
     set -x
 fi
+set -x
```

Comment:
Can you revert this change ? The block just above already allow to set it by setting a "DEBUG" env var to any value before calling the script :).